drivers/hud_apr/app/models/hud_apr/generators/shared/fy2020/question_six.rb

```diff
@@ -393,13 +393,13 @@ def self.question_number
 
       answer = @report.answer(question: table_name, cell: 'B4')
       members = stayers_with_anniversary.where(
-        a_t[:income_date_at_start].eq(nil).
-          or(a_t[:income_date_at_start].not_eq(a_t[:first_date_in_program])).
-          or(a_t[:income_from_any_source_at_start].in([nil, 8, 9])).
-          or(a_t[:income_from_any_source_at_start].eq(0).
-            and(income_jsonb_clause(1, a_t[:income_sources_at_start].to_sql))).
-          or(a_t[:income_from_any_source_at_start].eq(1).
-            and(income_jsonb_clause(1, a_t[:income_sources_at_start].to_sql, negation: true))),
+        a_t[:income_date_at_annual_assessment].eq(nil).
+          or(a_t[:annual_assessment_in_window].eq(false)).
+          or(a_t[:income_from_any_source_at_annual_assessment].in([nil, 8, 9])).
```

Comment:
Does this correctly turn into `income_from_any_source_at_annual_assessment in (8, 9) or income_from_any_source_at_annual_assessment is null`?

Reply:
Good catch! `leavers.where(a_t[:income_from_any_source_at_exit].eq(nil)).count != leavers.where(a_t[:income_from_any_source_at_exit].in([nil])).count)`